### DISCUSSION TITLE
［提问］如何更改GitHub头像

### DISCUSSION THREAD

**Discussion by Bahajsjsh** [share121/inter-knot#3365] [其他]
![emotion_download_1724168338040](https://github.com/user-attachments/assets/81ee3abf-2b09-4cbf-a341-3ff452caa6fd)

**Comment by yangsaho** [comment]
点击左上角的头像，之后再点击你的头像，右边会有一个Profile picture，点击下面的Edit就可以选你头像了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by Bahajsjsh** [reply]
谢了
